tests/integration_tests/test_examples/example_data/sample.ttl

```diff
@@ -0,0 +1,16 @@
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
```

Comment:
You can't just include something in the repo with no attribution... I'm also not 100% sure what is the exact license for W3C Recommendation examples. Can you instead put here some dummy file?